example/lib/neseted-realtime-events.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:async';
```

Comment:
should the file name be 
`nested...` instead of
`neseted...`?

Reply:
This file does not seem to be used anywhere. Is this intentional?

Reply:
Thanks for the typo find 😅 
I've moved some code that was being used for testing inside example app to this file to make `example/main.dart` a bit cleaner.

Reply:
shouldn't there be a button to resolve the conversation?

Reply:
file renamed 8dff0397450cb085fdf75dc5635c8d99290e824d
status comment added c61e063ef001e6d2aaa3d81b76666c903b169fe9

Reply:
I can see a "Resolve conversation" button.